app/services/gov_one_auth_service.rb

```diff
@@ -1,12 +1,17 @@
 # Service for interacting with Gov One Login
+# This service is initialised with an authorisation code and can be used to:
+# - exchange an authorisation code for tokens (access and id)
+# - exchange an access token for user info
+# - decode an id token to get the user's gov one id
+#
+# More information on the Gov One Login integration environment can be found here:
+# https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/
```

Comment:
Do you like the look of it in YARD? There is a `@see` tag you could try on the link. We can chat about it.

Reply:
Lovely!